src/components/modal/index.web.tsx

```diff
@@ -0,0 +1,110 @@
+// import _ from 'lodash';
+import React, {Component} from 'react';
+import {createPortal} from 'react-dom';
+import {ModalProps as RNModalProps, GestureResponderEvent, KeyboardAvoidingViewProps} from 'react-native';
+import {BlurViewPackage} from '../../optionalDependencies';
+import {asBaseComponent} from '../../commons/new';
+import TopBar, {ModalTopBarProps} from './TopBar';
+import View from '../../components/view';
+
+const BlurView = BlurViewPackage?.BlurView;
```

Comment:
the BlurView package doesn't exist on the web as far as I know, so you don't need it here